tests/rptest/tests/upgrade_test.py

```diff
@@ -614,3 +614,37 @@ def cloud_log_size() -> int:
         # Assert manifest has not been mutated by comparing sizes
         total_cloud_size_after_upgrade = cloud_log_size()
         assert total_cloud_size_before_upgrade <= total_cloud_size_after_upgrade, f"Mismatch in cloud storage size after upgrade, bytes before: {total_cloud_size_before_upgrade} bytes_after: {total_cloud_size_after_upgrade}"
+
+
+class InstallSelfTest(RedpandaTest):
+    @cluster(num_nodes=1)
+    def test_redpandainstaller_install(self):
+        """
```

Comment:
nit: could you add a comment mentioning that this isn't actually doing upgrades in the traditional sense, and instead is intentionally wiping and restarting the cluster with new binaries?